### PR TITLE
Rework README

### PR DESCRIPTION
**Pull request by Chadowo**
The [current README](https://github.com/HellRok/Taylor/blob/f2ce4024efc28a82a65ab5f10769310d114d94af/README.md) is nice, but can be improved, both in contents and visually (a tiny bit more visually) IMHO.  This PR aims to address that.

## Changes

The most notable changes can be found right at the beginning:

---
image
---

1. The title now is the Taylor logo, centered (the outline is for legibility on white theme).
2. The badges now all follow the flat round style, plus there are two new ones.
3. The links are now hyperlinks, separated by pipes.

Other minor changes include the new section at the the end that indicates the project's license (kinda standard for most projects) and  some really minimal text changes and grammar fixes.

The build section is the most affected by the changes of all sections, since IMO the older one sounds kinda stiff and hard to follow. I'd love to hear your thoughts on it!

## Logo

To get the logo for the README, I had to recreate it. It would be nice to have some sort of press kit for use in splash screens, title, website and other mediums. Like [Love2D](https://love2d.org/wiki/L%C3%B6ve_Logo_Graphics)'s or [Defold](https://defold.com/logo-and-trademark/)'s. I don't want to burden you with more tasks or anything but it's something nice that we could definitively do down the road :).

## Some Inspiration 

Here are some cool project READMEs that I've used for inspiration (taken from [here](https://github.com/matiassingers/awesome-readme)):

- [electron-markdownify](https://github.com/amitmerchant1990/electron-markdownify#readme).
- [dbt-core](https://github.com/dbt-labs/dbt-core#readme).
- [NSGIF](https://github.com/NSRare/NSGIF#readme).

### CODE REVIEW

**Comment by HellRok** [comment]
This looks absolutely fantastic! I love the changes to the build section, I didn't even consider using the `detail` element for hiding the long parts.

I'm wondering if we can use [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) as I'd like if the text could be fully black when in light mode but still fully white in dark mode as what you have looks awesome.
A picture of the readme in lightmode showing the text as just outlined

Thank you again for your hard work, your PRs have really helped bring Taylor to the next level!

**Comment by Chadowo** [comment]
Thanks for the feedback!

> I'm wondering if we can use [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) as I'd like if the text could be fully black when in light mode but still fully white in dark mode as what you have looks awesome.

[Seems like we're in luck](https://github.blog/changelog/2022-05-19-specify-theme-context-for-images-in-markdown-beta/), I'll do the change promptly.

> Thank you again for your hard work, your PRs have really helped bring Taylor to the next level!

NP, happy to help :)

**Comment by Chadowo** [comment]
@HellRok I changed some text in the README, along with the change you mentioned about dynamically changing the logo based on the user theme (f9909e0). If you're cool with how it currently then I think it would be ready for merging.

**Comment by Chadowo** [comment]
Also, [here](https://github.com/HellRok/Taylor/files/14555008/taylor-logos.tar.gz) are the files for the logos I've made.

**Review by HellRok — APPROVED** [review]

**Comment by HellRok** [comment]
Thanks for this, it looks amazing!
